ceph-salt-formula/salt/ceph-salt/apply/software.sls

```diff
@@ -6,6 +6,7 @@ install required packages:
   pkg.installed:
     - pkgs:
       - iputils
+      - iperf
```

Comment:
I know this will be needed in minions, but for consistency, can we also add this to the spec file just to make the requirement more explicit? 
(We are already doing the same for other dependencies)